src/middlewared/middlewared/plugins/sysdataset.py

```diff
@@ -433,7 +435,7 @@ async def __umount(self, pool, uuid):
 
         for dataset, name in reversed(self.__get_datasets(pool, uuid)):
             try:
-                await run('umount', '-f', dataset)
+                await run('umount', '-l', dataset)
```

Comment:
It will prevent pool from being exported. Whoever had the logs opened there for some reason will be logging to `/dev/null` It will be 10 times harder to debug.

Reply:
We don't care on enterprise HA systems because if the pool doesn't export within 5 seconds then we hard reset the node.

Reply:
The same code will be executed on non-HA systems, where we can't tell user "you need to reboot your NAS in order to export a pool".

Reply:
Then we have a design issue

Reply:
Because as it stands right now, we're breaking SCALE HA in favor of non HA users. So something else is going to have to be done.